Makefile

```diff
@@ -52,5 +52,6 @@ clean:
 .PHONY: dev
 dev:
 	go get github.com/pilu/fresh
-	docker-compose up
+	docker-compose start
```

Comment:
That's just a change back to the way it was before. I don't claim to understand docker-compose.